src/components/Immersion/ImmersionPresentationList.js

```diff
@@ -51,11 +51,13 @@ function ImmersionPresentationList({ actions, isLoading, items }) {
                           <td className="px-6 py-4 text-fv-charcoal">
                             {english}
                           </td>
-                          <td className="text-right px-6">
+                          <td className="text-right px-6" aria-label="list">
                             <ActionsMenu.Presentation
-                              docId={id}
-                              docTitle={immersionLabel}
-                              docType="label"
+                              entry={{
+                                docId: id,
```

Comment:
The property name here will need to be consistent with the property names you are accessing in Actions i.e. `id`, `title`, and `type`